xlive/H2MOD/Modules/Networking/CustomPackets/CustomPackets.cpp

```diff
@@ -162,13 +164,13 @@ void __stdcall message_gateway_hook_2(void *thisx, int a2, int message_type, int
 {
 	switch (message_type)
 	{
-	case e_network_message_types::request_map_filename:
+	case request_map_filename:
 	{
 		s_request_map_filename* pak = (s_request_map_filename*)packet;
-		LOG_TRACE_NETWORK("[H2MOD-CustomPackets] received on hook2 request-map-filename from XUID: {:#x}", pak->user_identifier);
+		LOG_TRACE_NETWORK("[H2MOD-CustomPackets] received on message_gateway_hook_2 request-map-filename from XUID: {:#x}", pak->user_identifier);
 		return;
 	}
-	case e_network_message_types::map_file_name:
+	case map_file_name:
 		return;
```

Comment:
so for map_file_name we don't want to call p_network_message_gateway_2 but for everything other type we do? 

there should be a single line doc explaining this and why